data_visualization_in_seaborn/data_visualization_seaborn.md

```diff
@@ -0,0 +1,684 @@
+</div>
+
+### Quiz: Line Plots
+
+True or False: You can create both scatter plots and line plots with the same function in `seaborn`.
+
+[(X)] TRUE
+[( )] FALSE
+****
+We used just one function, `relplot`, to make both scatter plots and line plots. Under the hood, there are actually two separate functions being used --- `scatterplot()` and `lineplot()`, which are equivalent to `relplot()` with `kind = "scatter"` (the default) and `kind = "line"`, respectively --- but it's nice to be able to stick to the higher-level `relplot` function for simplicity.
+
+Because they're both called by the `relplot` function, you can use the same additional variables (e.g. `hue`, `style`, `col`) applied to either scatter or line plots, and they work in much the same way.
+****
+
+True or False: Line plots are appropriate in most situations in which a scatter plot would work, so you can general pick whichever one you prefer.
+
+[( )] TRUE
+[(X)] FALSE
+****
+There are very few situations in which a scatter plot and line plot are equally appropriate.
+
+Line plots emphasize continuity and because of that are rarely used except for with data that have some kind of time component. Scatter plots are much more flexible and can work to show the relationship between almost any two continuous variables.  
+****
+
+What argument do you add to include facets in a `seaborn` plot?
+
+[[col, row]]
+<script>
+  let input = "@input".trim();
+  /col|row/i.test(input);
+</script>
+****
+You can add facets to a plot by including the arguments for `col`, `row`, or both.
+****
+
+## Trend Lines
+
+Trend lines look like line plots, but they are different in one key way: They show a **summary** of other data (usually a linear model), rather than plotting data directly.
+
+Trend lines are used to show the overall trend in a scatterplot. Sometimes, the scatterplot points themselves are omitted and just the trend lines are shown to keep the visualization as clean as possible, but they're still implied.
+
+### Linear regression trend lines
+
+Because trend lines are such a useful visual summary, `seaborn` provides several options to add trend lines to your plots quickly and easily.
+
+<div class = "options">
+There are two very similar functions available for drawing trend lines: `regplot` and `lmplot`. For many use cases, they are close to identical. For simplicity, we'll stick to `lmplot` in this lesson. Try subbing in `regplot` instead and make note of how the plots differ.
+
+For more background on `regplot` vs. `lmplot`, see [the seaborn regression tutorial](https://seaborn.pydata.org/tutorial/regression.html#controlling-the-size-and-shape-of-the-plot).
+</div>
+
+```python
+sns.lmplot(data = covid_data,
+            x="val_age", y="val_height_cm")
+```
+
+![Scatterplot with age on the x-axis and height on the y-axis, and a straight trend line running through the points, with a lighter shaded band around it.](media/seaborn_trend_1.png)
```

Comment:
more context/description of what's actually communicated via the image(e.g., mention the trend line fit to underlying data which you get into later)